frontend/src/helpers/API.js

```diff
@@ -32,34 +32,42 @@ axiosInstance.interceptors.response.use(
   (response) => response,
   // eslint-disable-next-line consistent-return
   (error) => {
-    const { status, message } = error.response;
-    if (status) {
-      switch (status) {
-        case 400:
-          alert(`${status}: ${message}`);
-          break;
-        case 401:
-          if (window.location.pathname !== '/login') {
-            if (currentSessionToken) {
-              axios.post('/token/delete');
-              sessionStorage.clear('token');
-            }
+    const errorResponse = error.response;
+    switch (errorResponse.status) {
+      case 400:
+        alert(`400: ${JSON.stringify(errorResponse.data)}`);
+        break;
+      case 401:
+        if (window.location.pathname !== '/login') {
+          // IMPORTANT: I WOULD NOT DO THIS!
```

Comment:
I think it would be better to put these comments on the PR where the code is introduced. Because we are not currently handling permissions this isn't a problem yet but should be documented as an issue for the backlog so we don't miss it.

Reply:
Good point. I will change this as we discussed.